crypto-core/src/main/java/com/palantir/crypto/cipher/AesCtrCipher.java

```diff
@@ -37,11 +35,11 @@
 public final class AesCtrCipher implements SeekableCipher {
 
     public static final String ALGORITHM = "AES/CTR/NoPadding";
-    static final String PROVIDER = "SunJCE";
     static final String KEY_ALGORITHM = "AES";
     static final int KEY_SIZE = 256;
     static final int BLOCK_SIZE = 16;
     static final int IV_SIZE = 16;
+    static byte[] out = new byte[BLOCK_SIZE];
```

Comment:
this looks like a potential place to run into some concurrency issues, I think either make this non-static (and either note that this class isn't thread-safe or mark the `seek` method synchronized) or make this `ThreadLocal`.